src/api/client.ts

```diff
@@ -103,7 +104,12 @@ function requestHandler(config: InternalAxiosRequestConfig): InternalAxiosReques
 async function requestIdTokenHandler(config: InternalAxiosRequestConfig) {
   if (config.authWithIdToken) {
     const user = auth().currentUser;
-    const idToken = await user?.getIdToken(config.forceRefreshIdToken);
+    const tokenResult = await user?.getIdTokenResult(config.forceRefreshIdToken);
+    if (config.forceRefreshIdToken && tokenResult?.claims['customer_number']) {
```

Comment:
@gorandalum @rosvik I was looking at and testing this change, however, if I remove `config.forceRefreshIdToken` to test the logic of the timer, it works it forces a retryAuth every 10 seconds. But `config.forceRefreshIdToken` is in the condition and testing on staging, the issue seems like the retryAut is not called, it could be because the `config.forceRefreshIdToken` is never set to true because the response did not get a 401 as a response instead might get something different because no internet.

Reply:
You need to use a proxy to introduce some 401 errors to test it appropriately.

Reply:
Yes, working on testing it on different types of responses.